src/pydicom/pixels/decoders/pyjpegls.py

```diff
@@ -18,16 +18,23 @@
 
 
 DECODER_DEPENDENCIES = {
-    uid.JPEGLSLossless: ("jpeg_ls>=1.1"),
-    uid.JPEGLSNearLossless: ("jpeg_ls>=1.1"),
+    uid.JPEGLSLossless: ("numpy", "pyjpegls>=1.2"),
+    uid.JPEGLSNearLossless: ("numpy", "pyjpegls>=1.2"),
```

Comment:
I'm confused now with "pyjpegls" vs "jpeg_ls" - how did this work before? And why it remains "jpeg_ls" below in the version check?

Reply:
This is just a label that gets used to notify people what dependencies/versions are missing. The actual import check is in `is_available()`